CONTRIBUTING.md

```diff
@@ -4,10 +4,9 @@ You can fork the repo and submit a pull request in Github.
 
 All developers must have signed the [P4.org](http://p4.org) CLA.
 
-### Madoko style checker
+### AsciiDoc style checker
 
 The P4Runtime specification is written using
-[Madoko](http://madoko.org/reference.html). We provide a lint tool to catch
+[AsciiDoc](https://docs.asciidoctor.org/). We provide a lint tool to catch
```

Comment:
I don't know of any lint tool for AsciiDoc source.  Do you?  If we don't have one, I think that is OK, but we should remove this text saying we have one.

Reply:
For the moment, I have removed it, but I plan to better understand the lint script provided in this repository